redaxo/src/addons/phpmailer/README.md

```diff
@@ -171,7 +179,18 @@ $mail->SMTPOptions = array(
 ```
 
 <a name="multidomain"></a>
-### Senden über unterschiedliche Domains
+### Senden über unterschiedliche Domains 
+
+Werden E-Mails über unterschiedliche Absender-Domains verschickt, sollte der SPF-Eintrag der Absender-Domain in den DNS-Einstellungen
```

Comment:
vllt hier nen link auf eine website die das thema SPF kurz umreist?

Reply:
Wikipedia hilft nicht viel. Es müsste was einfaches sein, ein Tutorial, denke ich. Einen Link zu einem Hoster möchte ich aber auch ungerne setzen. Die haben häufig nützliche Anleitungen. 
Übernehme das erstmal so und ich mache es später in der Doku ausführlicher?